doc/reference/cmdline.rst

```diff
@@ -6,282 +6,297 @@
 Command-line interface: odoo.py
 ===============================
 
-.. _reference/cmdline/server:
+.. _reference/cmdline/env:
 
-Running the server
-==================
+Installing Development Environment
+==================================
 
-.. program:: odoo.py
+    .. note:: This section applies to those developing within
+              a linux environment.
 
-.. option:: -d <database>, --database <database>
+One option for bootstrapping an Odoo 9 development environment goes as follows:
+    .. code-block:: console
 
-    database used when installing or updating modules.
+        $ wget -O- https://raw.githubusercontent.com/odoo/odoo/9.0/odoo.py | python
 
-.. option:: -i <modules>, --init <modules>
 
-    comma-separated list of modules to install before running the server
-    (requires :option:`-d`).
+The preceding command will download the Odoo 9 branch, via wget; configure git,
+such as finding the appropriate remotes; and setup Odoo's dependencies, including Postgres.
 
-.. option:: -u <modules>, --update <modules>
 
-    comma-separated list of modules to update before running the server
-    (requires :option:`-d`).
+.. _reference/cmdline/scaffold:
 
-.. option:: --addons-path <directories>
+Creating a Module Skeleton
```

Comment:
Small section and so I thought to 'just get it out of the way'... You're right though... Will move to the very bottom...